<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roi</class>
 <widget class="QMainWindow" name="roi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>roi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="connectBtn">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnectBtn">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="acceptBtn">
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>img1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>img2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>img3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Database">
    <property name="title">
     <string>New Database</string>
    </property>
    <addaction name="actionOpen_Database"/>
    <addaction name="action_Accept"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_Disconnect"/>
   </widget>
   <addaction name="menuNew_Database"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Database"/>
   <addaction name="actionOpen_Database"/>
   <addaction name="actionAdd_Item"/>
   <addaction name="actionRemove_Item"/>
   <addaction name="action_Close"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Database">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="action_Accept">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/accept.png</normaloff>:/accept.png</iconset>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/close.png</normaloff>:/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/cloud_disconnection.png</normaloff>:/cloud_disconnection.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionAdd_Item">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/add_item.png</normaloff>:/add_item.png</iconset>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
   <property name="toolTip">
    <string>Add Item</string>
   </property>
  </action>
  <action name="actionRemove_Item">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/remove_item.png</normaloff>:/remove_item.png</iconset>
   </property>
   <property name="text">
    <string>Remove Item</string>
   </property>
   <property name="toolTip">
    <string>Remove Item</string>
   </property>
  </action>
  <action name="actionNew_Database">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new_database.png</normaloff>:/new_database.png</iconset>
   </property>
   <property name="text">
    <string>New Database</string>
   </property>
   <property name="toolTip">
    <string>New Database</string>
   </property>
  </action>
  <action name="action_Connect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/cloud_connection.png</normaloff>:/cloud_connection.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
